### PR TITLE
Enabling Publish Code Coverage Results V2 task and adding L0 tests

### PR DESCRIPTION
**Pull request by vinayakmsft**
**Task name**: Publish Code Coverage Results V2

**Description**: Added the new task in the make-options.json. Added unit tests to verify the functionalities. 

**Documentation changes required:** (Y/N) Y . We will have to create a new documentation for PublishCodeCoverageResultsV2 task while releasing it for public

**Added unit tests:** (Y/N) Y . Following are the screenshots of the passed L0 test cases. 

image
image

** UI and logs of the task enabled in private account:**
MicrosoftTeams-image
MicrosoftTeams-image (1)
MicrosoftTeams-image (2)

**Attached related issue:** (Y/N) N
** General Note**: The answer.ts file is similar to the one that was present in PublishCodeCoverageResultV1 task. It is required for validation. 
**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: make-options.json [hunk 1]
```diff
@@ -151,6 +151,7 @@
         "PowerShellOnTargetMachinesV3",
         "PublishBuildArtifactsV1",
         "PublishCodeCoverageResultsV1",
+        "PublishCodeCoverageResultsV2",
```
Comment (triptijain2112) [inline make-options.json:154]: Please update task version to latest:
image

Comment (vinayakmsft) [reply make-options.json:154]: updated

**Review by triptijain2112 — COMMENTED** [review]
Nit: coverage.cobertura (4) (1).xml file name and please update "FilesForTesting" folder name

**Review by triptijain2112 — APPROVED** [review]
Approving the PR on behalf of @Eswarnath.

**Pull request merged**